docs/Deployment/BackupRestore.md

```diff
@@ -0,0 +1,324 @@
+
+The original strategy of backups in BTCPay Server still exists and can be found [here](https://docs.btcpayserver.org/Docker/#how-can-i-back-up-my-btcpay-server).
+While we did create a new process, the old scripts still do work and have only slightly been touched. See [pull request #641](https://github.com/btcpayserver/btcpayserver-docker/pull/641) for more details. 
+
+:::tip 
+Please be aware of these important issues:
+The old channel state is toxic, and you can lose all your funds if you or someone else closes a channel based on the backup with the old state - and the state changes often! If you publish an old state (say from yesterday's backup) on-chain, you will most likely lose all your funds in the channel because the counterparty might publish a [revocation transaction](https://www.d11n.net/lightning-network-payment-channel-lifecycle.html#what-happens-in-case-of-a-false-close%3F)!
+:::
+
+The backup process gets started from the `btcpay-backup.sh` script. 
+Open a command prompt and type the following. 
+
+```
+cd "$BTCPAY_BASE_DIRECTORY"
+./btcpay-backup.sh
+```
+A bash script will start running your backup when running the above command. 
+Below we will talk you through the steps taken. 
+The backup process needs to be started as root, which it will check for and let you know if you have to switch users.
+After the user has been determined suitable, the script will set itself up to take the next steps:
+
+* checks for docker dir.
+* Makes a dump of the postgres DB. 
+* Sets BTCPay directory. 
+* Sets backup directory. 
+* Determines the DB dump path. 
+* Determines the backup path. 
+
+If the backup directory doesn't exist yet, the script will ensure to make one. 
+With these preparations taken, the backup process is now starting. 
+
+The script has checks to ensure it either works or fails with a comprehensive error message at every step of the way.
+This gets determined by a `-e` tag at the start of the script. 
+
+```
+echo "🚨 Database container could not be started or found."
+```
+
+If everything is running smoothly, you'll get to see multiple completed marks in your console. 
+Whenever the backup has been completed successfully, it will state:
+
+```
+printf "✅ Backup done => $backup_path\n\n"
+```
+
+Your BTCPay Server has now finished the backup process. It's now up to you to store these backups in a safe manner. 
+After you've made a backup the first time, it's always wise to at least test your backup in a restore scenario. 
+In the next topic, we will go over the extra options you can set with your backup.
+
+## Extra options for `btcpay-backup.sh`
+
+When you run the `btcpay-backup.sh` script, you're able to set certain flags.
+
+### Set a Passphrase for BTCPay Server backup. 
+
+One of the extra tags that you can set when running the `btcpay-backup.sh` is for a passphrase.
+This would be run as follows :
+
+```
+cd "$BTCPAY_BASE_DIRECTORY"
+./btcpay-backup.sh BTCPAY_BACKUP_PASSPHRASE 
```

Comment:
you just put the passphrase behind the backup script? Without any option flag introducing it? Seems a bit weired for me. Also the passphrase would now exist in the shell(bash)'s history file. Might be worth mentioning?

Reply:
I've now mentioned it in the text, but I think we should solve it using another way, like using an env file, so that it doesn't end up in the history.